core/modules/filter/filter.admin.inc

```diff
@@ -152,6 +152,9 @@ function filter_admin_format_form($form, &$form_state, $format) {
   $form['#submit'] = array();
   $form['#tree'] = TRUE;
   $form['#attached']['library'][] = array('filter', 'filter.admin');
+  $form['#attached']['js'][] = backdrop_get_path('module', 'filter') . '/js/filter.admin.js';
+  $form['#attached']['css'][] = backdrop_get_path('module', 'filter') . '/css/filter.css';
+  backdrop_add_library('system', 'ui.dialog');
```

Comment:
Let's be consistent here and use:
```
$form['#attached']['library'][] = array('system', 'ui.dialog');
```

Or better, use the Backdrop dialog library to help match the existing Backdrop styling:
```
$form['#attached']['library'][] = array('system', 'backdrop.dialog');
```